packages/pluggableWidgets/area-chart-web/src/AreaChart.tsx

```diff
@@ -0,0 +1,70 @@
+import classNames from "classnames";
+import { createElement, ReactElement, useCallback } from "react";
+import { AreaChartContainerProps, SeriesType } from "../typings/AreaChartProps";
+import { ChartWidget, ChartWidgetProps } from "@mendix/shared-charts";
+import { getPlotChartDataTransforms, SeriesMapper, usePlotChartDataSeries } from "@mendix/shared-charts/hooks";
+// import { getPlotChartDataTransforms, usePlotChartDataSeries } from "@mendix/shared-charts/hooks";
```

Comment:
commented duplication

Reply:
As there is no duplication anymore i am solving this discussion